apps/web/src/components/TrendingSuggestions/TrendingSuggestions.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { render, waitFor, screen } from '@testing-library/react';
+import TrendingSuggestions from './TrendingSuggestions';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+const queryClient = new QueryClient();
+
+const server = setupServer(
+  rest.get('trending-suggestions', (req, res, ctx) => {
+    return res(
+      ctx.json([
+        {
+          URL: 'https://images.unsplash.com/photo-1515879218367-8466d910aaa4?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTB8fGNvZGV8ZW58MHx8MHx8&w=1000&q=80',
```

Comment:
Same comment as above about considering reusing one previously defined variable within this array since they are all the same. Actually in this case it would be better to use a function that accepts the heading number argument.